app/components/tag.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react'
+import clsx from 'clsx'
+import type {MouseEventHandler} from 'react'
+
+interface TagProps {
+  tag: string
+  selected?: boolean
+  onClick?: MouseEventHandler<HTMLButtonElement>
+}
+
+function Tag({tag, selected, onClick}: TagProps) {
+  // TODO: make this accessible, aria-checked and aria-selected aren't allowed on buttons
+  //   should we make it a checkbox with @reach/checkbox?
```

Comment:
:point_up_2:

Reply:
Yes, let's do that 👍

Reply:
done :sunglasses: 

![tags](https://user-images.githubusercontent.com/1196524/123936695-bd0c8180-d995-11eb-8faf-a976e5b9573e.gif)